docs/pages/world/modules.mdx

```diff
@@ -132,13 +134,11 @@ import { mudConfig } from "@latticexyz/config";
 export default mudConfig({
   tables: {
     MyTable: {
-      name: "mytable",
       schema: "uint32",
     },
   },
   modules: [
     {
-      name: "KeysWithValueModule",
```

Comment:
Modules still have names.
Pls typecheck stuff in your editor before changing the doc. We need to be careful to not put stuff that doesn't work.

Reply:
Typechecking in editor is a good idea thanks. The imports for the config were actually wrong.